### DISCUSSION TITLE
Question regarding exploit via a race condition

### DISCUSSION THREAD

**Discussion by Tarun5139** [AdnaneKhan/Gato-X#54] [Q&A]
Hello @AdnaneKhan , 
How can i exploit 
Could brief me more about it 
![WhatsApp Image 2024-10-12 at 8 15 47 PM](https://github.com/user-attachments/assets/2d69d801-6a0a-4d1e-91b9-9812fdad1047)

**Comment by AdnaneKhan** [comment]
This is not the appropriate please to ask if something is exploitable or not. Please read about how GitHub Actions works and determine that for yourself.
